src/components/select/multi-select/multi-select.component.js

```diff
@@ -493,24 +493,27 @@ const MultiSelect = React.forwardRef(
       <StyledSelectMultiSelect
         aria-expanded={isOpen}
         aria-haspopup="listbox"
-        ref={containerRef}
         disabled={disabled}
         readOnly={readOnly}
         hasTextCursor
         size={size}
         data-component={dataComponent}
         data-role={dataRole}
         data-element={dataElement}
+        isOpen={isOpen}
         {...filterStyledSystemMarginProps(textboxProps)}
       >
-        <SelectTextbox
-          aria-controls={isOpen ? selectListId.current : ""}
-          type="text"
-          labelId={labelId.current}
-          positionedChildren={disablePortal && isOpen && selectList}
-          {...getTextboxProps()}
-        />
-        {!disablePortal && isOpen && selectList}
+        <div ref={containerRef}>
```

Comment:
The containerRef will no longer contain the selectList, why did this change?

Reply:
Thats intentional because the SelectList now renders an overlay that takes the space of the whole screen, so clicking on the background would not close the Select without this change